simulation-runner/src/main/scala/com/puppetlabs/gatling/node_simulations/FOSS5xPerfMedium.scala

```diff
@@ -13,48 +13,44 @@ import io.gatling.http.Predef._
 class FOSS5xPerfMedium extends SimulationWithScenario {
 
 // 	val httpProtocol = http
-// 		.baseURL("https://puppetserver-perf-sut56.delivery.puppetlabs.net:8140")
+// 		.baseURL("https://puppetserver-perf-sut57.delivery.puppetlabs.net:8140")
+// 		.acceptHeader("application/json, text/pson")
+// 		.acceptEncodingHeader("gzip;q=1.0,deflate;q=0.6,identity;q=0.3")
+// 		.userAgentHeader("Puppet/5.0.0 Ruby/2.4.1-p111 (x86_64-linux)")
 
-	val reportBody = ELFileBody("FOSS5xPerfMedium_0127_request.txt")
+	val reportBody = ElFileBody("FOSS5xPerfMedium_0127_request.txt")
 
-	val headers_0 = Map(
-		"X-Puppet-Version" -> "5.0.0",
-		"Accept" -> "application/json",
-		"Accept-Encoding" -> "gzip;q=1.0,deflate;q=0.6,identity;q=0.3")
-// val uri1 = "https://puppetserver-perf-sut56.delivery.puppetlabs.net:8140/puppet/v3"
-
-	val headers_report = Map(
-		"X-Puppet-Version" -> "5.0.0",
-		"Accept" -> "application/json",
+	val baseHeaders = Map("Accept" -> "application/json, text/pson",
 		"Accept-Encoding" -> "gzip;q=1.0,deflate;q=0.6,identity;q=0.3",
-		"Content-Type" -> "application/json",
-		"Connection" -> "close")
+		"User-Agent" -> "Puppet/5.0.0 Ruby/2.4.1-p111 (x86_64-linux)")
 
-	val headers_catalog = Map(
-		"X-Puppet-Version" -> "5.0.0",
-		"Accept" -> "application/json",
-		"Accept-Encoding" -> "gzip;q=1.0,deflate;q=0.6,identity;q=0.3",
-		"Content-Type" -> "application/x-www-form-urlencoded")
+	val headers_0 = baseHeaders ++ Map("X-Puppet-Version" -> "5.0.0")
+
+	val headers_127 = baseHeaders ++ Map(
+		"Connection" -> "close",
+		"Content-Type" -> "application/json",
+		"X-Puppet-Version" -> "5.0.0")
+// val uri1 = "https://puppetserver-perf-sut57.delivery.puppetlabs.net:8140/puppet/v3"
 
 	val chain_0 = exec(http("node")
-			.get("/puppet/v3/node/${node}?environment=production&transaction_uuid=9198e795-dd6f-4599-a3b6-4b927651272e&fail_on_404=true")
+			.get("/puppet/v3/node/${node}?environment=production&transaction_uuid=6646b2f9-ddcb-485d-aec0-b7687659d7ff&fail_on_404=true")
 			.headers(headers_0))
-		.pause(127 milliseconds)
+		.pause(101 milliseconds)
 		.exec(http("filemeta pluginfacts")
 			.get("/puppet/v3/file_metadatas/pluginfacts?environment=production&links=follow&recurse=true&source_permissions=use&ignore=.svn&ignore=CVS&ignore=.git&ignore=.hg&checksum_type=md5")
 			.headers(headers_0))
-		.pause(351 milliseconds)
+		.pause(355 milliseconds)
 		.exec(http("filemeta plugins")
 			.get("/puppet/v3/file_metadatas/plugins?environment=production&links=follow&recurse=true&source_permissions=ignore&ignore=.svn&ignore=CVS&ignore=.git&ignore=.hg&checksum_type=md5")
 			.headers(headers_0))
-		.pause(1)
+		.pause(2)
 		.exec(http("catalog")
 			.post("/puppet/v3/catalog/${node}?environment=production")
-			.headers(headers_catalog)
+			.headers(headers_0)
```

Comment:
I confirmed when playing the recording attached to this PR via gatling that a `Content-Type: application/x-www-form-urlencoded` header was sent in the request.  So I think we're good here.